dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentletAPIImpl.java

```diff
@@ -281,7 +281,8 @@ public Contentlet findContentletByIdentifier(String identifier, boolean live, lo
                 throw new DotContentletStateException("No contenlet found for given identifier");
             }
             if(live){
-                return find(clvi.getLiveInode(), user, respectFrontendRoles);
+                return (com.dotmarketing.util.UtilMethods.isSet(clvi.getLiveInode()))?
+                        find(clvi.getLiveInode(), user, respectFrontendRoles): null;
```

Comment:
what about places like this one? they are expecting the exception thrown here. I think this would fail if changed to return null instead